.github/workflows/webext.yml

```diff
@@ -18,6 +18,9 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
+      - name: Install rename
+        run: sudo apt install rename
```

Comment:
sudo apt update is needed before that line .

Reply:
Indeed, thanks!